packages/solid/package.json

```diff
@@ -112,6 +112,7 @@
   "dependencies": {
     "@uploadthing/dropzone": "workspace:*",
     "@uploadthing/shared": "workspace:*",
+    "solidjs-use": "2.3.0",
```

Comment:
Is this maintained?

Repo's archived: https://github.com/solidjs-use/solidjs-use

Reply:
I guess not if it is archived... :/ I can try going back to the old way, I think the only real issue was binding to window instead of document, but i didn't find that until later.